tests/xfstests/install.pm

```diff
@@ -36,7 +36,7 @@ sub run {
 
     # Add QA repo
     my $qa_head_repo = get_var('QA_HEAD_REPO', '');
-    zypper_call("--no-gpg-check ar -f '$qa_head_repo' qa-ibs", timeout => 600);
+    script_run("zypper --no-gpg-check ar -f '$qa_head_repo' qa-ibs", 600);
```

Comment:
Why is this needed?

Reply:
In my tests, the zypper_call() fails because qa-ibs already exists.

Reply:
OK, but in that case I don't understand why `script_run` will solve your issue, as it will do the same and try to add an already existing package. `-f` option should do the job and reinstall the qa-ibs package.

Reply:
`script_run` won't return 1 if the command fails.  The command is installing a new repository, not a new package.

Reply:
Yes, indeed it's `ar`, my bad :-).

But I don't really like this approach, as `zypper_call` **should** be used for zypper calls. And mainly because this test is not dedicated to our team.

Maybe could be better to test if the repo is already installed and only do this if needed?